### DISCUSSION TITLE
Multiple Ports and machine names per container

### DISCUSSION THREAD

**Discussion by beautiful-orca** [almeidapaulopt/tsdproxy#25] [Q&A] [answered]
Can i expose multiple ports of a container under seperate subdomain?

For example if i have a page on port 8000 and an page on port 9000 of the same container. I want https://page1.ts.net to serve port 8000 and https://page2.ts.net to serve port 9000.

**Comment by SolidRhino** [comment]
I hope this function will be implemented. 😊

**Comment by beautiful-orca** [comment, marked as answer]
It can be done with proxy lists function. It works like a reverse proxy configuration, you set your subdomain (tailnet device name) to forward to a ip (or localhost) and port

https://almeidapaulopt.github.io/tsdproxy/docs/list/

**Comment by almeidapaulopt** [comment]
it's already implemented in a very early stage
